### PR TITLE
Add i18n to server-side messages

### PR DESCRIPTION
**Pull request by int-y1**
Affected pages:

- `/admin/judge/contest/1/change/` > Problems > Rejudge column
- `/admin/judge/judge/add/` > Regenerate button
- `/admin/judge/submission/` > Rejudge button (and Locked button, if sub is locked)
- `/contest/c1/participations/<user>` page's title
- Too many running submissions
- Submit to an unviewable problem

### CODE REVIEW

File: judge/admin/submission.py [hunk 1]
```diff
@@ -244,9 +244,10 @@ def language_column(self, obj):
 
     def judge_column(self, obj):
         if obj.is_locked:
-            return format_html('<input type="button" disabled value="Locked"/>')
+            return format_html('<input type="button" disabled value="{0}"/>', _('Locked'))
         else:
-            return format_html('<input type="button" value="Rejudge" onclick="location.href=\'{}/judge/\'" />', obj.id)
+            return format_html('<input type="button" value="{0}" onclick="location.href=\'{1}/judge/\'" />',
```
Comment (Ninjaclasher) [inline judge/admin/submission.py:249]: While we're changing this code, could we update this to reverse the URL instead of this hack?

Comment (int-y1) [reply judge/admin/submission.py]: Done.

File: judge/admin/contest.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ class ContestProblemInline(SortableInlineAdminMixin, admin.TabularInline):
     def rejudge_column(self, obj):
         if obj.id is None:
             return ''
-        return format_html('<a class="button rejudge-link" href="{}">Rejudge</a>',
+        return format_html('<a class="button rejudge-link" href="{1}">{0}</a>', _('Rejudge'),
```
Comment (Xyene) [inline judge/admin/contest.py:76]: Could we flip the arg order here?

Comment (int-y1) [reply judge/admin/contest.py]: Done. The button is still good.

Capture

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**